test/release/examples/users-guide/base/constParam.chpl

```diff
@@ -1,4 +1,5 @@
 use IO;
+import Math.pi;
```

Comment:
This line won't show up in the user's guide, so I'd probably just do `use IO, Math;` to avoid mixing `use` and `import` in a single program (which seems like an obvious "Why did they do that?" kind of stumbling block.  But also because—the way the users guide works today—it refers to line numbers to determine which code excerpts to include, so changing the lines by adding new ones requires changing the references to them as well.  By not creating/destroying lines, you won't have to worry about breaking the cross-references.  (Yes, this is annoyingly fragile; hopefully we can use the much better blog framework to avoid this in the next draft).